src/containers/settings/pluginSetting/component.tsx

```
@@ -58,111 +96,142 @@ class SettingDialog extends React.Component<
+                      toast.error(
+                        this.props.t("Only desktop version supports TTS plugin")
+                      );
+                      return;
+                    }
+                    if (
+                      plugin.type === "voice" &&
+                      plugin.voiceList.length === 0
+                    ) {
+                      let voiceFunc = plugin.script;
+                      // eslint-disable-next-line no-eval
+                      eval(voiceFunc);
```

Biome (2.4.7)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)